### ISSUE TITLE
Luminosity ratio for focus Indicator of 'File' button is 1.3:1, which is less than 3:1.

### ISSUE THREAD

**Issue by msftedad** [tinymce/tinymce#8906]
**What is the current behavior? Describe the bug**
The luminosity ratio of the focus indicator of the 'File' button present in toolbar is 1.3:1 which is less than 3:1

**Please provide the steps to reproduce and if possible a minimal demo of the problem via [fiddle.tiny.cloud](https://fiddle.tiny.cloud/) or similar.**
1. Open URL: https://codepen.io/Roopesh-T/pen/ZEmbPBK
2. Navigate to 'Text edit field'.
3. Navigate to tool bar present above 'Text edit field' using alt + F10 key
4. Navigate to 'File' button.
5. Turn on color contrast analyser tool.
6. Verify whether luminosity ratio of focus indicator for 'File' button is more than 3:1 or not.

**What is the expected behavior?**
The luminosity ratio of focus indicator for 'File' button should be more than 3:1.

**Which versions of TinyMCE, and which browser / OS are affected by this issue? Did this work in previous versions of TinyMCE?**

**Comment by sumenaaz** [comment]
@Afraithe :What is the time frame your team is targeting to fix this issue? Could you please provide an estimated date?

**Comment by msftedad** [comment]
Hi team, Any update on this issue?

**Comment by msftedad** [comment]
Hi team, Any update on this issue?

**Comment by msftedad** [comment]
Hi team, Any update on this issue?

**Comment by msftedad** [comment]
Hi Team, As issue is still repro. Could you please provide update if there is any?
